### PR TITLE
Remove logging level configuration

### PR DESCRIPTION
**Pull request by haosong**
Address #44

### CODE REVIEW

File: wavefront_sdk/common/heartbeater_service.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
 from .constants import SERVICE_TAG_KEY
 from .constants import SHARD_TAG_KEY
 
-logging.basicConfig(level=logging.INFO)
 LOGGER = logging.getLogger(__name__)
```
Comment (ustinov) [inline wavefront_sdk/common/heartbeater_service.py]:
```suggestion
```

File: wavefront_sdk/common/heartbeater_service.py [hunk 2]
```diff
@@ -91,7 +88,7 @@ def _report(self):
         # pylint: disable=broad-except,fixme
         # TODO: Please make sure we catch more specific exception here.
         except Exception:
-            LOGGER.warning('Can not report %s to wavefront', HEART_BEAT_METRIC)
+            logging.warning('Can not report %s to wavefront', HEART_BEAT_METRIC)
```
Comment (ustinov) [inline wavefront_sdk/common/heartbeater_service.py]:
```$ python -m flake8

./wavefront_sdk/common/heartbeater_service.py:91:80: E501 line too long (80 > 79 characters)
```

**Comment by ustinov** [comment]
What about these?
```
$ grep -r LOGGER .
./wavefront_sdk/proxy.py:LOGGER = logging.getLogger('wavefront_sdk.WavefrontDirectClient')
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/proxy.py:            LOGGER.warning("Can't send data to Wavefront. Please configure "
./wavefront_sdk/common/metrics/registry.py:LOGGER = logging.getLogger('wavefront_sdk.WavefrontSdkMetricsRegistry')
./wavefront_sdk/common/metrics/registry.py:                LOGGER.warning('Unable to send internal SDK metric.')
```

**Comment by haosong** [comment]
@ustinov Might need to pull the latest

**Pull request merged**